src/sentry/newsletter/remote.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import absolute_import
+
+import logging
+from uuid import uuid4
+from hashlib import sha1
+
+from sentry.http import safe_urlopen
+from sentry.newsletter.base import Newsletter
+from sentry.utils import json
+
+logger = logging.getLogger('sentry.newsletter')
+
+
+def get_install_id():
```

Comment:
isnt this guaranteed to exist from runner? if not maybe we should enforce it there